boilerplate/app/i18n/ar.json

```diff
@@ -27,6 +27,7 @@
   "demoNavigator": {
     "componentsTab": "عناصر",
     "debugTab": "تصحيح",
+    "listTab": "TODO: List",
```

Comment:
Should we change this to `MAVERICKTODO` ?

Reply:
ya jamon what the heck?

Reply:
![CALLED OUT](https://c.tenor.com/pR11uP8q98kAAAAd/madagascar-skipper.gif)